components/SpotInfo/index.js

```diff
@@ -7,49 +7,35 @@ import {
   InformationWrapper,
   Longitude,
   Latitude,
-  EntryTextarea,
-  NoInfoMessage,
-  InformationForm,
   EntryList,
-  InfoTextarea,
-  InfoLabel,
-  InfoCreateButtonWrapper,
-  InfoCreateButton,
+  EntryCard,
+  EntryTextarea,
+  EntryDeleteButton,
+  StyledIcon,
+  NoEntryMessage,
+  AddEntryForm,
+  AddEntryLabel,
+  AddEntryTextarea,
+  AddEntryButtonWrapper,
+  AddEntryButton,
   SpotDeleteButton,
 } from "./style";
 import LoadingSpinner from "../LoadingSpinner";
 import Error from "../Error";
+import { PiTrash, PiPencilLight } from "react-icons/pi";
 
 export default function SpotInfo({ spotId }) {
-  const {
-    data: spot,
-    error,
-    isValidating,
-  } = useSWR(`/api/spots/${spotId}`);
+  const { data: spot, error, isValidating } = useSWR(`/api/spots/${spotId}`);
```

Comment:
If you want to revalidate your cache, destructure `mutate` right here. And then use it below as `mutate()` only. Remove the import of `mutate` at the top level of this file.